yarn.lock

```diff
@@ -30900,7 +31457,7 @@ winston@^3.0.0, winston@^3.3.3:
     triple-beam "^1.3.0"
     winston-transport "^4.4.0"
 
-winston@^3.5.1:
+winston@^3.4.0, winston@^3.5.1:
   version "3.5.1"
   resolved "https://registry.yarnpkg.com/winston/-/winston-3.5.1.tgz#b25cc899d015836dbf8c583dec8c4c4483a0da2e"
   integrity sha512-tbRtVy+vsSSCLcZq/8nXZaOie/S2tPXPFt4be/Q3vI/WtYwm7rrwidxVw2GRa38FIXcJ1kUM6MOZ9Jmnk3F3UA==
```

Comment:
It looks like `winston@^3.4.0, winston@^3.5.1` and `winston@^3.0.0, winston@^3.3.3` above can be collapsed into a single dependency.